ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -141,7 +160,7 @@ function mirror_release_image_for_disconnected_upgrade() {
     # All IPv6 clusters are disconnected and
     # release image should be mirrored for upgrades.
     if [[ "${DS_IP_STACK}" == "v6" ]]; then
-      echo "### Mirroring release image for disconnected upgrade ###"
+      echo "### Mirroring release images for disconnected upgrade ###"
 
       MIRROR_RESULT=$(run_mirror_release_image_for_disconnected_upgrade_ssh_commands || echo "fail")
```

Comment:
to make debugging slightly easier, can you use:

```sh
echo "JUnit failing result written to ${JUNIT_IMAGE_FILE}"
```

and similar in the echos, so we don't have to drop into the JUnits to see if they're successes or failures?